bin/methods/install.js

```diff
@@ -0,0 +1,22 @@
+const cwd = process.cwd()
+const log = require('log-cb')
+const npm = require('npm')
```

Comment:
I'll check for that.. I just updated this pull request with automatic plugin detection from `./plugins/**/index.js` where ** is the plugin name for example `cordlr-role`.

could you check it? It's pretty bad written since it's strange how the global globbing handles it. maybe you can find a way to refactor that?

otherwise it's working for now